src/main/scala/com/monsanto/arch/cloudformation/model/resource/ApiGateway.scala

```diff
@@ -0,0 +1,260 @@
+package com.monsanto.arch.cloudformation.model.resource
+
+import com.monsanto.arch.cloudformation.model.{ConditionRef, ResourceRef, Token, `Fn::GetAtt`}
+import spray.json.{DefaultJsonProtocol, JsonFormat}
+import DefaultJsonProtocol._
+
+case class `AWS::ApiGateway::Account`(
+                                       name: String,
+                                       CloudWatchRoleArn: Option[String] = None,
+                                       override val Condition: Option[ConditionRef] = None
+                                     ) extends Resource[`AWS::ApiGateway::Account`] {
+  override def when(newCondition: Option[ConditionRef]) = copy(Condition = newCondition)
+}
+object `AWS::ApiGateway::Account` {
+  implicit val format: JsonFormat[`AWS::ApiGateway::Account`] = jsonFormat3(`AWS::ApiGateway::Account`.apply)
+}
+
+case class `AWS::ApiGateway::ApiKey`(
+                                      name: String,
+                                      Description: Option[String] = None,
+                                      Enabled: Option[Boolean] = None,
+                                      Name: Option[String] = None,
+                                      StageKeys: Option[Seq[StageKey]] = None,
+                                      override val Condition: Option[ConditionRef] = None
+                                    ) extends Resource[`AWS::ApiGateway::ApiKey`] {
+  def apiKey: Token[String] = ResourceRef(this)
+
+  override def when(newCondition: Option[ConditionRef]) = copy(Condition = newCondition)
+}
+object `AWS::ApiGateway::ApiKey` {
+  implicit val format: JsonFormat[`AWS::ApiGateway::ApiKey`] = jsonFormat6(`AWS::ApiGateway::ApiKey`.apply)
+}
+
+case class StageKey(RestApiId: Option[String] = None, StageName: Option[String] = None)
```

Comment:
updated in https://github.com/MonsantoCo/cloudformation-template-generator/pull/89/commits/756377475449f58dd9a3e3a2fbcbd3f3e125b409